wire/internal/wire/parse.go

```diff
@@ -193,8 +194,14 @@ type Value struct {
 // Load finds all the provider sets in the given packages, as well as
```

Comment:
Can you clarify the relationship between "given packages" and `patterns`? Same in `load`.

Reply:
Done.